lib/Mojo/Util.pm

```diff
@@ -794,6 +794,8 @@ Create anonymous scope guard object that will execute the passed callback when t
 
 Constant time comparison algorithm to prevent timing attacks.
 
+The secret string should be the second argument, to avoid leaking information about the length of the string.
```

Comment:
This is still inconsistently formatted.

Reply:
Apologies. I don't understand how you want it formatted. Is there a style guide somewhere, or am I missing something obvious here?

Reply:
Just look at the surrounding documentation. There is no newline between sentences.